TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/CapturedTrafficToHttpTransactionAccumulator.java

```diff
@@ -239,6 +242,9 @@ private boolean  rotateAccumulationOnReadIfNecessary(String connectionId, Accumu
     private boolean handleEndOfRequest(Accumulation accumulation) {
         assert accumulation.state == Accumulation.State.NOTHING_SENT : "state == " + accumulation.state;
         var requestPacketBytes = accumulation.rrPair.requestData;
+        metricsLogger.atSuccess()
+                .addKeyValue("requestId", accumulation.getRequestId())
```

Comment:
This is a UniqueRequestKey.  How will the composite object be emitted?  This feels like you REALLY should be doing this with a connectionId key rather than something truly unique.  I'm expecting questions like, what's the distribution of # of captured requests per connection id.  With the requestId in there too, everything would be unique.

Reply:
UniqueRequestKeys are emitted nicely as strings, e.g. `"0242acfffe140008-0000000e-00000005-d8a1533aa2d1dee1-3888e983.0"`

Reply:
Don't ever rely upon toString().  It's used by many different callers with different needs.  Tomorrow, it could be changed to ",0" or the longChannelId portion could be truncated.  If you need something specific, make the code state that.